src/firebase-db.ts

```diff
@@ -102,6 +102,13 @@ export async function initializeDB(name: FirebaseAppName) {
     await firebase.firestore().disableNetwork();
   }
 
+  // See: https://concord-consortium.slack.com/archives/C0303QUNU/p1604596779407700
```

Comment:
Do we have a policy on linking to internal CC slack conversations from public GitHub repos? Seems like it would be better to copy the relevant content from the slack conversation so that outsiders aren't confused when they can't access it.